tests/dotnet/chart/templates/deployment.yaml

```diff
@@ -40,8 +39,9 @@ kind: Service
 metadata:
   name: test-app-dotnet-service
 spec:
-  type: LoadBalancer
+  type: NodePort
```

Comment:
For unknown reasons, I had to use `LoadBalancer` here to get the .NET tests to work locally. Maybe this works now, I don't know.

Reply:
I checked and there's no issues with NodePort vs LoadBalancer, as long as you add `/WeatherForecast` to the URL that the Makefile tries to open since it doesn't know about paths and tries to open `/`.